src/renderer/browser/util/parseURL.js

```diff
@@ -1,7 +1,12 @@
-import { trim } from 'lodash';
+import { filter, isEmpty, trim, trimEnd, endsWith } from 'lodash';
+
+import { HTTP, CUSTOM_PROTOCOLS } from '../values/protocols';
 
 export default function parseURL(query) {
-  const trimmedQuery = trim(query);
-  const formattedQuery = trimmedQuery.includes('://') ? trimmedQuery : `nos://${trimmedQuery}`;
-  return new URL(formattedQuery);
+  const trimmedQuery = trimEnd(trim(query), '/').split('://').pop();
+
+  const tld = filter(CUSTOM_PROTOCOLS, (protocol) => endsWith(trimmedQuery, protocol.tld));
+  const tldToApply = isEmpty(tld) ? HTTP : tld[0].protocol;
+
```

Comment:
There is a very minor issue with this where the user could enter "https://foo.neo", but we redirect to "nos://foo.neo".    It's only exacerbated because we're splitting on all `://` and not just the first.  I think we can resolve this pretty easily though.  I'll throw up an extra commit of what I think this looks like so we can get this merged.